src/components/__tests__/TeamSeas.Install.test.js

```diff
@@ -1,11 +0,0 @@
-/* eslint-env jest */
```

Comment:
This file had just one test which is duplicated in the other TeamSeasInstall component test file, so I just deleted this.